Documentation for Partial and Downstream Pipelines in CI Visibility (#15726)
DataDog/documentation: content/en/continuous_integration/pipelines/gitlab.md

**Note**: The Pipelines page shows data for only the default branch of each repository.

### Partial and downstream pipelines

> In the `Pipeline Executions` page, you can now use the filters below in the search bar:

Reviewer: ```suggestion
In the **Pipeline Executions** page, you can use the filters below in the search bar:
```

Reviewer: Same question as the other doc regarding this screenshot

> * `Downstream Pipeline` - possible values: `true`, `false` 
> * `Manually Triggered` - possible values: `true`, `false` 
> * `Partial Pipeline` - possible values: `retry`, `paused`, `resumed`

Reviewer: ```suggestion
`Downstream Pipeline`
: Possible values: `true`, `false`

`Manually Triggered`
: Possible values: `true`, `false`

`Partial Pipeline`
: Possible values: `retry`, `paused`, `resumed`
```

Reviewer: Suggesting to incorporate this content as a description list for added clarity.

> {{< img src="ci/partial_retries_search_tags.png" alt="GitLab Partial Retry Filters" style="width:100%;">}}

Reviewer: ```suggestion
{{< img src="ci/partial_retries_search_tags.png" alt="The Pipeline executions page with Partial Pipeline:retry entered in the search query" style="width:100%;">}}
```

> These filters can also be applied via the facet panel on the left hand side of the page.

Reviewer: ```suggestion
These filters can also be applied through the facet panel on the left hand side of the page.
```

> {{< img src="ci/partial_retries_facet_panel.png" alt="GitLab Partial Retry Facet Panel" style="width:100%;">}}

Reviewer: ```suggestion
{{< img src="ci/partial_retries_facet_panel.png" alt="The facet panel with Partial Pipeline facet expanded and the value Retry selected, the Partial Retry facet expanded and the value true selected" style="width:40%;">}}
```

Reviewer: It looks like this screenshot would also benefit from a lower width setting

Author: As long as the text in the screenshot is not too small to see!

**Note**: The Pipelines page shows data for only the default branch of each repository.

> ### Partial and downstream pipelines
> In the `Pipeline Executions` page, you can now use the filters below in the search bar:

Reviewer: ```suggestion
### Partial and downstream pipelines

In the **Pipeline Executions** page, you can now use the filters below in the search bar:

```

Author: No worries, thank you!
